pillarbox-monitoring-grafana/provisioning/dashboards/dashboard.yml

```
@@ -0,0 +1,12 @@
+apiVersion: 1
+
+providers:
+  - name: 'Player QoS'
```

Maybe

```suggestion
  - name: 'Player Monitoring'
```

would be nicer?